templates/web/base/front/site-message.html

```diff
@@ -1,7 +1,11 @@
 [% site_message = c.cobrand.site_message %]
 [% IF site_message && !rendered_site_message %]
 <div class="site-message">
+[% IF site_message.match('<[a-z][\s\S]*>', 'i') %]
```

Comment:
There are some other places this would be good - `report/new/form_after_heading.html` for the reporting message, base/peterborough `waste/header.html` for waste, and `templates/web/bathnes/around/_postcode_form_post.html`. Perhaps a new Filter in `Template.pm` that does this check and then calls html_paragraph or not, so it's hidden from the templates, could just have `site_message | html_para_if_no_html` or something like that?